Gruntfile.js

```diff
@@ -87,12 +87,16 @@ module.exports = function(grunt) {
         bgShell: {
             ember: {
                 cmd: 'npm run build -- --watch',
-                bg: true,
-                stdout: function (out) {
-                    grunt.log.writeln(chalk.cyan('Ember-cli::') + out);
-                },
-                stderror: function (error) {
-                    grunt.log.error(chalk.red('Ember-cli::' + error));
+                bg: true
+            }
+        },
+
+        watch: {
+            csscomb: {
+                files: ['app/styles/**/*.css'],
+                tasks: ['shell:csscombfix'],
+                options: {
+                    livereload: 35730
```

Comment:
The reason why this port is specified is because the default port would conflict with the watch task of the server repo when running `grunt dev`